main/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"os"
+
+	"github.com/google/subcommands"
+)
+
+func main() {
+	subcommands.Register(subcommands.HelpCommand(), "")
+	subcommands.Register(subcommands.FlagsCommand(), "")
+	subcommands.Register(subcommands.CommandsCommand(), "")
+	subcommands.Register(&interdiffCmd{}, "")
```

Comment:
It would be common to have this in the init() function in the relevant file. https://golang.org/doc/effective_go.html#init - Even with a single package, each file can have its own init function. Doing the registration of interdiffCmd and mixedCmd in the init functions means that  the main file doesn't need knowledge of which subcommands are available and adding/removing a subcommand will only touch a single file.